lib/shopify-cli/packager.rb

```diff
@@ -0,0 +1,81 @@
+module ShopifyCli
+  class Packager
+    def initialize
+      @packaging_dir = File.join(ShopifyCli::ROOT, 'packaging')
```

Comment:
Could these be constants? e.g. `PACKAGING_DIR = File.join(ShopifyCli::ROOT, 'packaging')`

Reply:
Absolutely! I keep forgetting that ruby constants can have code in them 🙂